batch/test/test_dag.py

```diff
@@ -304,3 +304,63 @@ def test_no_parents_allowed_without_batches(client):
         assert re.search('.*invalid parent batch: .*', err.response.text)
         return
     assert False
+
+
+def test_output_files_no_service_account_is_error(client):
+    batch = client.create_batch()
+    try:
+        batch.create_job('alpine:3.8',
+                         command=['/bin/sh', '-c', 'echo head > /out'],
+                         output_files=[('/out', 'gs://hail-ci-0-1-batch-volume-test-bucket')])
+    except requests.exceptions.HTTPError as err:
+        assert err.response.status_code == 400
+        assert re.search('.*invalid request: service account may be specified '
+                         'if and only if input_files or output_files is set.*',
```

Comment:
Do you mean `service account must be specified`

Reply:
Hmm. "X may be true if and only if Y is true" sounds right to me. I suppose must also reads fine? I'm not really sure.